README.md

```diff
@@ -1 +1,171 @@
-# vercel-edge
+# Eppo Javascript SDK for Vercel
```

Comment:
Love having all the info for our SDK in it's README (as opposed to just a link to our documentation)